pkg/globalmanager/controllers/incrementallearning/downstream.go

```diff
@@ -76,16 +76,20 @@ func (c *Controller) syncToEdge(eventType watch.EventType, obj interface{}) erro
 
 	var trainNodeName string
 	var evalNodeName string
+	var deployNodeName string
 
 	ann := job.GetAnnotations()
 	if ann != nil {
 		trainNodeName = ann[runtime.AnnotationsKeyPrefix+string(sednav1.ILJobTrain)]
 		evalNodeName = ann[runtime.AnnotationsKeyPrefix+string(sednav1.ILJobEval)]
+		if _, ok := ann[runtime.ModelHotUpdateAnnotationsKey]; ok {
```

Comment:
why not just 
`deployNodeName = ann[runtime.AnnotationsKeyPrefix+string(sednav1.ILJobDeploy)]` like above lines?

Reply:
fix it